actions/port-forward/action.yaml

```diff
@@ -24,15 +28,16 @@ runs:
         version: ${{ inputs.kubectl-version }}
 
     - name: Establish port forward
-      shell: bash
+      env:
+        TIMEOUT: "${{ inputs.timeout }}"
       run: |
         kubectl port-forward "services/${{ inputs.service }}" "${{ inputs.port }}:${{ inputs.port }}" --namespace ${{ inputs.namespace }} --kubeconfig ~/.kube/config > port-forward.out &
         # wait up to 60 seconds for port forwarding
-        for _ in {1..120}; do if grep -q "Forwarding from 127.0.0.1:${{ inputs.port }} -> ${{ inputs.port }}" port-forward.out; then
+        for _ in {1..$TIMEOUT}; do if grep -q "Forwarding from 127.0.0.1:${{ inputs.port }} -> ${{ inputs.port }}" port-forward.out; then
           echo "Port forwarding done"
           break
         else
-          cat port-forward.out || true
```

Comment:
Where did this line go? This was usesful for seeing errors from the port-forward